jobs/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class Jobs(models.Model):
+    company_name = models.CharField(max_length=100, blank=True, null=True)
+    position = models.CharField(max_length=100, blank=True, null=True)
+    location = models.CharField(max_length=100, blank=True, null=True)
+    entry_timestamp = models.DateTimeField(auto_now=True)
+    unique_id = models.CharField(max_length=255, null=True, unique=True)
```

Comment:
If `unique_id` uniquely identifies a job then better to make it a primary key too and call it `job_id` etc

Reply:
There could be a case while scrapping data from website we do not get any values for position, location or company , in that case unique_id would be blank/null that's why I didn't use it as a primary key.